app/views/registration_wizard/start.html.erb

```diff
@@ -37,6 +37,10 @@
       Before registering for an NPQ you’ll be asked to create a DfE Identity account or sign in to your account.
     </p>
 
+    <p class="govuk-body">
+      By continuing, you agree to our <%= govuk_link_to("privacy notice", "https://www.gov.uk/government/publications/privacy-information-education-providers-workforce-including-teachers/privacy-information-education-providers-workforce-including-teachers#NPQ") %>.
```

Comment:
Would be great to have that as constant in `config/initializers/constants.rb` :)

Reply:
Using constant for the url now.